document/Conventions.mdk

```diff
@@ -4,6 +4,11 @@
 
 The following conventions are used within this document.
 
+## Key words for requirements levels
+
+In this document, the key words "must", "must not", "required", "shall", "shall not",
+"should", "should not", "recommended", "may", and "optional" are used as defined in RFC 2119.
```

Comment:
>RFC 2119 [](start = 84, length = 8)

link to it? [https://www.ietf.org/rfc/rfc2119.txt](https://www.ietf.org/rfc/rfc2119.txt)